packages/dripsy/src/core/types-v2/sx.ts

```diff
@@ -199,6 +203,8 @@ const sx: Sx = {
   bg: '$text',
   padding: '$1',
   m: '$1',
+  w: '$1',
```

Comment:
i would have this as width not w if it works. its less confusing

Reply:
Note, that this object has a test-only purpose, and`w` is added here to test that newly added aliases work as espected (as well as already existed aliases like `m` and `bg`).

Reply:
True I over looked I’m fine with it either way